app.js

```diff
@@ -1,36 +1,59 @@
+import fs from 'fs';
 import bodyParser from 'body-parser';
+import json from 'express';
```

Comment:
What is this for exactly?

Reply:
It's a leftover import, I tried using `app.use(json())` instead of the (according to my linter) deprecated `bodyParser.json()` but couldn't get it to work. I'll remove this import